src/parser/error.rs

```diff
@@ -108,12 +108,10 @@ impl ParseError {
     pub fn report<'a>(
         &self,
         source: &'a str,
-        origin: Option<&'a str>,
+        origin: &std::string::String,
         colored: bool,
         ascii: bool,
     ) -> std::io::Result<String> {
-        let origin = origin.unwrap_or("input");
-
```

Comment:
please revert this change, we want `Option<&str>`, even tho we now just unwrap it to `input`, there's plans to change that into something else in the future.

Reply:
Fixed